sycl/test-e2e/SpecConstants/2020/kernel-bundle-api.cpp

```diff
@@ -12,6 +12,7 @@
 // FIXME: ACC devices use emulation path, which is not yet supported
 // UNSUPPORTED: accelerator
 // UNSUPPORTED: hip
+// UNSUPPORTED: native_cpu
```

Comment:
Why is this test being marked as unsupported on native CPU if this PR is intended to add support for this feature? Does it mean that the test passed and this patch introduces a regression?

Reply:
We run Native CPU `e2e` tests downstream, we weren't running the tests in `test-e2e/SpecConstants` before this PR. With this PR all the tests in `test-e2e/SpecConstants` are passing, besides this one, which is also marked as `UNSUPPORTED` for other targets that use emulated SC, so this is not a regression.